docs/drill4net/support-matrix.mdx

```diff
@@ -0,0 +1,54 @@
+﻿---
+id: support-matrix
+title: Support Matrix
+---
+import { Image } from '@components';
+import { Info } from '@components';
+import { Table } from '../../src/components'
+
+## Testing Frameworks
+
+<Table>
+
+| Framework |      Supported versions     |
+|:---------:|:---------------------------:|
+| NUnit     |	    - 3.13.3			  |
+| XUnit     | 2.4.2 - 4.13.2              |
+| Selenium  |	    - 4.12.4			  |
+| Playwright|		- 1.32.0			  |
+
+</Table>
+
+## .NET Frameworks
+
+<Table>
+
+| Framework |      Supported versions     |
+|:---------:|:---------------------------:|
+| .NET			 | 5, 6, 7			   	  |
+| .NET Core      | 1.0 - 3.1              |
```

Comment:
Please specify intermediary versions - the exact list. Aavoid using `-` since it is ambiguous

Reply:
Same applies for other tables